test_outputs/connlist/udn_test_1_tcp-90_explain_output.txt

```diff
@@ -0,0 +1,52 @@
+
+#######################################################
+# Specific connections and their reasons in blue[udn] #
+#######################################################
+----------------------------------------------------------------------------------------------------------------------------------------------------------------
+Connections between 0.0.0.0-255.255.255.255[External] => blue[udn]/webserver[Pod]:
+
+Allowed connections:
+	Allowed TCP:[90] due to the system default (Allow all)
+
+----------------------------------------------------------------------------------------------------------------------------------------------------------------
+Connections between blue[udn]/webserver[Pod] => 0.0.0.0-255.255.255.255[External]:
+
+Allowed connections:
+	Allowed TCP:[90] due to the system default (Allow all)
+
+----------------------------------------------------------------------------------------------------------------------------------------------------------------
+Connections between blue[udn]/webserver[Pod] => green[udn]/webserver[Pod]:
```

Comment:
why is this pair (mixing two udns) under `blue[udn]` header?

Reply:
it was a random decision to put it under the "src's udn"; in the mean while a connection which mixes two udns may appear in the report only in case we are using a "focus-*" flag to run the command 
(there are no allowed conns between peers in different networks, and denied cross network connection will not appear in the report unless using a specific focu-conn/workload flag)